README.md

```diff
@@ -4,7 +4,7 @@ Symfony Demo Application
 The "Symfony Demo Application" is a reference application created to show how
 to develop Symfony applications following the recommended best practices.
 
-[![Build Status](https://travis-ci.org/symfony/symfony-demo.svg?branch=master)](https://travis-ci.org/symfony/symfony-demo)
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy) [![Build Status](https://travis-ci.org/symfony/symfony-demo.svg?branch=master)](https://travis-ci.org/symfony/symfony-demo)
```

Comment:
The "original" idea of the button is to live in repos' `README`s. The trick is that with a regular `heroku.com/deploy` link, the Heroku dashboard reads the repo from the `Referer` (sic) header, so if someone forks the "symfony-demo" repo, that fork is also instantly deployable without edits, including branches, which is quite nice for PRs etc, because the corresponding branch can be deployed by anyone as well.

Having it on symfony.com/demo would of course be fantastic ;) but it's a separate thing.